Userland/Libraries/LibSymbolication/Symbolication.cpp

```diff
@@ -61,24 +61,41 @@ Optional<FlatPtr> kernel_base()
 
 Optional<Symbol> symbolicate(String const& path, FlatPtr address)
 {
-    if (!s_cache.contains(path)) {
-        auto mapped_file = MappedFile::map(path);
-        if (mapped_file.is_error()) {
-            dbgln("Failed to map {}: {}", path, mapped_file.error().string());
+    String full_path = path;
+    if (!path.starts_with('/')) {
+        const char* search_paths[] = { "/usr/lib/{}", "/usr/local/lib/{}" };
```

Comment:
How about using `StringView`s? (Maybe even `Array<StringView>`)

Reply:
The code now uses an `Array<StringView, 2>`